lib/python/temporal/core.py

```diff
@@ -113,7 +113,7 @@ def get_tgis_database():
 # can differ this value must be an integer larger than 0
 # Increase this value in case of backward incompatible changes
 # temporal database SQL layout
-tgis_db_version = 2
+tgis_db_version = 3
```

Comment:
Does this have any negative impact on backward compatibility with existing TGRASS DBs? I don't expect it, but just asking...

Reply:
Yes, unfortunately this means that `t.*` modules will be not able to work with DB created by `t.create` with this PR applied.

```
G76> t.info test
ERROR: Unsupported temporal database: version mismatch.
...
Supported temporal database version is: 2
```